kedro/framework/session/session.py

```diff
@@ -72,7 +72,7 @@ class KedroSessionError(Exception):
 class KedroSession:
     """``KedroSession`` is the object that is responsible for managing the lifecycle
     of a Kedro run.
-    - Use `KedroSession.create("<your-kedro-project-package-name>")` as
+    - Use `KedroSession.create("<package_name>")` as
     a context manager to construct a new KedroSession with session data
     provided (see the example below).
     - Use `KedroSession(session_id=<id>)` to instantiate an existing session with a given
```

Comment:
Unrelated to your change, but this docstring is out od date. Let's remove the second bullet point on "`KedroSession(session_id=<id>)`" completely. So the first bullet point can then become un-bulleted also.

Reply:
Changes made in [5bd959c](https://github.com/kedro-org/kedro/pull/1753/commits/5bd959c450f25a10a400d32a89ba128559fe2382).